### PR TITLE
[@types/oracledb] Fix FetchTypeResponse

### PR DESCRIPTION
**Pull request by ghost**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

- [X] Provide a URL to documentation or source code which provides context for the suggested changes: (https://node-oracledb.readthedocs.io/en/latest/api_manual/oracledb.html#oracledb.fetchTypeHandler)
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Comment by dannyb648** [comment]
@sla100  what is the purpose of the change? Does the whitespace cause an issue?

File: types/oracledb/index.d.ts [hunk 1]
```diff
@@ -560,6 +560,7 @@ declare namespace OracleDB {
     function converter<T>(arg: T): T;
 
     type FetchTypeResponse =
+        | undefined
         | { type: DbType; converter?: typeof converter }
         | { type?: DbType; converter: typeof converter };
```
[the remark this answers is not in the document]
Comment (sla100) [reply types/oracledb/index.d.ts]: Sure. I changed it.

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Review by diogopxrnextday — APPROVED** [review]

**Comment by dannyb648** [comment]
@sla100 sorry for the delay, Weve already for a fix for this in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69245/files

Appreciate your help!

**Comment by MysteryBlokHed** [comment]
> @sla100 sorry for the delay, Weve already for a fix for this in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69245/files
> 
> Appreciate your help!

It looks like the linked PR only changed one function, while this PR changed two. Is that intended?

**Comment by sla100** [comment]
@MysteryBlokHed
You're right. The second PR must be extended.

**Review by gabritto — APPROVED** [review]

**Comment by sla100** [comment]
Ready to merge

**Pull request merged**